### PR TITLE
Feature/se-54

### LINKED ISSUE #54: Create the front-end for LinedIn Learning explain Video
a Linked in learning

### PR DESCRIPTION
**Pull request by Chrissydrx**
close SE2-WS23/documentation#54
image
I added a basic LinkedIn Learning page. Currently, the video player does nothing.

### CODE REVIEW

File: src/main.jsx [hunk 1]
```diff
@@ -40,6 +41,10 @@ const router = createBrowserRouter([
     element: <LoginPage />,
     errorElement: <ErrorPage />,
   },
+  {
+    path: "/linked-in-learning",
+    element: <LinkedInLearning />,
+  },
```
Comment (bishop6232) [inline src/main.jsx]: everything looks good but the only thing I found to be missing is the error handling element for the route you created.

**Review by bishop6232 — CHANGES_REQUESTED** [review]

File: src/pages/LinkedInLearning/LinkedInLearningPage.jsx
Comment (bishop6232) [inline src/pages/LinkedInLearning/LinkedInLearningPage.jsx:1]: everything looks good and well structured 👍

File: src/pages/LinkedInLearning/LinkedInLearningPage.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      <Box
+        sx={{
+          display: "flex",
+          justifyContent: "center",
+          alignItems: "center",
+          height: "calc(100% - 60px)",
+        }}
+      >
+        <video width="80%" height="90%" controls>
+          <source src="" type="video/mp4" />
```
Comment (julix14) [inline src/pages/LinkedInLearning/LinkedInLearningPage.jsx]: Why didn't you implemented directly a way to pass the video source as a prop into the component?

File: src/main.jsx (hunk 1 above)
Comment (Chrissydrx) [reply src/main.jsx]: I restructured the whole route structure. Is this also fine for you?

File: src/pages/LinkedInLearning/LinkedInLearningPage.jsx (hunk 2 above)
Comment (Chrissydrx) [reply src/pages/LinkedInLearning/LinkedInLearningPage.jsx]: Changed

File: src/main.jsx
Comment (bishop6232) [inline src/main.jsx:1]: Yes, this looks much better and clean 👍

**Review by bishop6232 — APPROVED** [review]
Thank you for the changes, it looks good.

**Pull request merged**